static/js/components/tiles/ranking_tile.tsx

```diff
@@ -54,6 +54,145 @@ interface RankingGroup {
   dateRange: string;
 }
 
+interface SvRankingUnitsProps {
```

Comment:
since a lot of these props can be derived from RankingData and RankingTileSpec, maybe lets just pass those in so there aren't so many props to keep track of ..

Reply:
done